.dockerignore

```diff
@@ -0,0 +1,9 @@
+.git
+.github
+.pytest_cache
+build
+dist
+**/*.zip
```

Comment:
This needs to be out: .zip files are used in the test-data, i.e. Shapefile .zips for OGR tests, or maybe make an exception for those data files.

Reply:
Thank you.
I would rather use `COPY`/`ADD` commands on specific files as much as possible and reduce the usage of `.dockerignore`. It is rather easy to end up with additional files when testing/developing locally which then gets added to the locally built image.

Reply:
Yes, makes sense!